pages/linux/rpcclient.md

```diff
@@ -0,0 +1,36 @@
+# rpcclient
+
+> MS-RPC client tool (part of the samba suite).
+> More information: <https://www.samba.org/samba/docs/current/man-html/rpcclient.1.html>.
+
+- Connect to a host:
+
+`rpcclient --user {{domain}}\{{username}}%{{password}} {{ip}}`
+
+- Connect to a host on a domain without a password:
+
+`rpcclient --user {{username}} --workgroup {{domain}} --no-pass {{ip}}`
+
+- Connect to a host, passing the password hash:
+
+`rpcclient --user {{domain}}\{{username}} --pw-nt-hash {{ip}}`
+
+- Enumerate domain users:
+
+`rpcclient $> enumdomusers`
+
+- Enumerate domain groups:
```

Comment:
Can we use the verb `display` or `list` to replace `enumerate`?

Reply:
OK, handled in the latest commit.